naga/src/front/spv/mod.rs

```diff
@@ -4221,6 +4216,102 @@ impl<I: Iterator<Item = u32>> Frontend<I> {
                     self.upgrade_atomics
                         .insert(ctx.get_contained_global_variable(p_exp_h)?);
                 }
+                Op::AtomicCompareExchange => {
+                    inst.expect(9)?;
+
+                    let start = self.data_offset;
+                    let span = self.span_from_with_op(start);
+                    let result_type_id = self.next()?;
+                    let result_id = self.next()?;
+                    let pointer_id = self.next()?;
+                    let _memory_semantics_id = self.next()?;
```

Comment:
(carried over) The SPIR-V spec calls this "Scope <id>: Memory". So maybe this should be called `_memory_scope_id`.